src/ui/templates/search/autocomplete.hbs

```diff
@@ -53,7 +53,7 @@
               data-short="{{shortValue}}"
               data-filter='{{json filter}}'
               data-eventtype="AUTO_COMPLETE_SELECTION"
-              data-eventoptions='{"suggestedSearchText": "{{value}}"}'
```

Comment:
For my edification, does handlebars just automatically remove escape characters? I tried on https://handlebarsjs.com/playground.html and it looks like it replaces them with `&quot'`s?

I was wondering if we could just use triple braces instead to fix this issue? it seeeems to work like we'd want but I haven't tested anything beyond this screenshot
![image](https://user-images.githubusercontent.com/23005393/160154179-b4996424-5420-49bf-9161-cfeba248102a.png)

Reply:
^I'm also not sure if this approach would have security issues so maybe just having the explicit `eventOptions` data is better?

Reply:
I attempted the triple curly braces (didn't work) before going with this approach. I do think having an explicit eventOptions data is better though

Reply:
Yea, when I logged out value from handlebars log helper in template, the escape characters were removed. someone tried to do something similar with JSON directly as string and it's [not recommended](https://github.com/handlebars-lang/handlebars.js/issues/1301)

Reply:
thanks for looking into it